installers/vmware/kickstart-esxi.go

```diff
@@ -383,6 +383,9 @@ func rootpw(j job.Job) string {
 // We do not support anything other than ESXi 6.5 and above (os slug "vmware_esxi_6_5", "vmware_esxi_6_7", "vmware_esxi_7_0" etc)
 // full list of drive settings is listed https://packet.atlassian.net/browse/SWE-2385
 func determineDisk(j job.Job) string {
+	if j.BootDriveHint() != "" && j.PlanSlug()[:1] == "s" {
```

Comment:
if `j.PlanSlug()` == "" then `j.PlanSlug()[:1]` will cause a panic. another option could be to use `strings.HasPrefix` which wont panic if `j.PlanSlug()` == "".

Reply:
In this test hardness that should never happen, but this is still very good feedback and a change worth incorporating.